Source/EasyNetQ.Management.Client.IntegrationTests/ManagementClientTests.cs

```diff
@@ -8,138 +8,138 @@
-			}
-			foreach (var listener in overview.Listeners)
-			{
-				Console.Out.WriteLine("listener.IpAddress = {0}", listener.IpAddress);
-			}
-
-			Console.Out.WriteLine("overview.Messages = {0}", overview.QueueTotals?.Messages ?? 0);
-
-			foreach (var context in overview.Contexts)
-			{
-				Console.Out.WriteLine("context.Description = {0}", context.Description);
-			}
-		}
-
-		[Fact]
-		public async Task Should_get_nodes()
-		{
-			var nodes = (await managementClient.GetNodesAsync().ConfigureAwait(false)).ToList();
-
-			nodes.Count.ShouldNotEqual(0);
-			nodes[0].Name.ShouldEqual($"rabbit@{rabbitHostName}");
-		}
-
-		[Fact]
-		public async Task Should_get_definitions()
-		{
-			var definitions = await managementClient.GetDefinitionsAsync().ConfigureAwait(false);
-
-			definitions.RabbitVersion[0].ShouldEqual('3');
-		}
-
-		[Fact]
-		public async Task Should_get_connections()
-		{
-			foreach (var connection in await managementClient.GetConnectionsAsync().ConfigureAwait(false))
-			{
-				Console.Out.WriteLine("connection.Name = {0}", connection.Name);
-
-				var clientProperties = connection.ClientProperties;
-
-				Console.WriteLine("User:\t{0}", clientProperties.User);
-				Console.WriteLine("Application:\t{0}", clientProperties.Application);
-				Console.WriteLine("ClientApi:\t{0}", clientProperties.ClientApi);
-				Console.WriteLine("ApplicationLocation:\t{0}", clientProperties.ApplicationLocation);
-				Console.WriteLine("Connected:\t{0}", clientProperties.Connected);
-				Console.WriteLine("EasynetqVersion:\t{0}", clientProperties.EasynetqVersion);
-				Console.WriteLine("MachineName:\t{0}", clientProperties.MachineName);
-
-				//Test the dynamic nature
-				Console.WriteLine("Copyright:\t{0}", ((dynamic)clientProperties).Copyright);
-			}
-		}
-
-		[Fact(Skip = "Requires at least a connection")]
-		public async Task Should_be_able_to_close_connection()
-		{
-			// first get a connection
-			var connections = await managementClient.GetConnectionsAsync().ConfigureAwait(false);
-
-			// then close it
-			await managementClient.CloseConnectionAsync(connections.First()).ConfigureAwait(false);
-		}
-
-		[Fact]
-		public async Task Should_throw_when_trying_to_close_unknown_connection()
-		{
-			var connection = new Connection {Name = "unknown"};
-			try
-			{
-				await managementClient.CloseConnectionAsync(connection).ConfigureAwait(false);
-			}
-			catch (Exception e)
-			{
-				Assert.IsType<UnexpectedHttpStatusCodeException>(e);
-			}
-		}
-
-		[Fact]
-		public async Task Should_get_channels()
-		{
-			var channels = await managementClient.GetChannelsAsync().ConfigureAwait(false);
-
-			foreach (var channel in channels)
-			{
-				Console.Out.WriteLine("channel.Name = {0}", channel.Name);
-				Console.Out.WriteLine("channel.User = {0}", channel.User);
-				Console.Out.WriteLine("channel.PrefetchCount = {0}", channel.PrefetchCount);
-			}
-		}
+    [Integration]
```

Comment:
please fix all this spaces? did you add tabs instead? please revert on what was before, to see only your changes

Reply:
maybe it's some spacing fixed after, BTW please resolve the conflict